docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -10,6 +11,214 @@ recommendations: false
 ---
 # Tutorial: Test a .NET class library with .NET using Visual Studio
 
+::: zone pivot="dotnet-6-0"
+
+This tutorial shows how to automate unit testing by adding a test project to a solution.
+
+## Prerequisites
+
+- This tutorial works with the solution that you create in [Create a .NET class library using Visual Studio](library-with-visual-studio.md).
+
+## Create a unit test project
+
+Unit tests provide automated software testing during your development and publishing. [MSTest](https://github.com/Microsoft/testfx-docs) is one of three test frameworks you can choose from. The others are [xUnit](https://xunit.net/) and [nUnit](https://nunit.org/).
+
+1. Start Visual Studio.
+
+1. Open the `ClassLibraryProjects` solution you created in [Create a .NET class library using Visual Studio](library-with-visual-studio.md).
+
+1. Add a new unit test project named "StringLibraryTest" to the solution.
+
+   1. Right-click on the solution in **Solution Explorer** and select **Add** > **New project**.
+
+   1. On the **Add a new project** page, enter **mstest** in the search box. Choose **C#** or **Visual Basic** from the Language list, and then choose **All platforms** from the Platform list.
+
+   1. Choose the **MSTest Test Project** template, and then choose **Next**.
+
+   1. On the **Configure your new project** page, enter **StringLibraryTest** in the **Project name** box. Then choose **Next**.
+
+   1. On the **Additional information** page, select **.NET 6.0 (Preview)** in the **Framework** box. Then choose **Create**.
```

Comment:
Is is still called .NET 6.0 (Preview) for RC 1?

Reply:
Good question, I assume it is but will change the tutorial if they change the UI.